src/spatialdata/datasets.py

```diff
@@ -75,7 +86,9 @@ def raccoon(
 class BlobsDataset:
     """Blobs dataset."""
 
-    def __init__(self, length: int = 512, n_points: int = 200, n_shapes: int = 5) -> None:
+    def __init__(
+        self, length: int = 512, n_points: int = 200, n_shapes: int = 5, extra_coord_space: Optional[str] = None
```

Comment:
I would call it `coord_system` and not space

Reply:
resolved with [bef3a35](https://github.com/scverse/spatialdata/pull/282/commits/bef3a353b97f515e5561d8672c88b55252b7a675)